website/pages/en/chain-integration-overview.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Chain Integration Process Overview
+---
+
+A transparent and governance-based integration process was designed for blockchain teams seeking integration with The Graph protocol. It is a 3-phase process, as summarised below.
+
+## Graph Chain Integration Process
```

Comment:
Is this title necessary? Because if it is, the `##`s below should be `###`s, but I would just remove it and then they can remain `##`s.

Reply:
No, not 100% necessary. Removing!